extension/src/plots/model/index.ts

```diff
@@ -96,10 +96,15 @@ export class PlotsModel extends ModelWithPersistence {
   }
 
   public async transformAndSetPlots(data: PlotsOutput, revs: string[]) {
-    this.fetchedRevs = new Set([...this.fetchedRevs, ...revs])
+    const cliIdToLabel = this.getCLIIdToLabel()
```

Comment:
[F] When we get the data back from the CLI we substitute the label for the short sha that we provided.